homeassistant/components/maxcube/binary_sensor.py

```diff
@@ -1,8 +1,10 @@
 """Support for MAX! binary sensors via MAX! Cube."""
 from homeassistant.components.binary_sensor import (
+    DEVICE_CLASS_BATTERY,
```

Comment:
Can you use the new BinarySensorDeviceClass enumerator?

Reply:
Done https://github.com/home-assistant/core/pull/58699/commits/cfb0f28ba70e049d29bb281e6c614404e0c5049a

Reply:
Also removed these now unused imports https://github.com/home-assistant/core/pull/58699/commits/a7f57f91da703274515e1f27611ad823e29b0bd2